payment-failure/src/main/scala/com/gu/identity/paymentfailure/Model.scala

```diff
@@ -1,15 +1,36 @@
 package com.gu.identity.paymentfailure
 
-import io.circe.generic.semiauto.deriveDecoder
-import io.circe.Decoder
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+
+case class Config(idapiHost: String, brazeApiHost: String, idapiAccessToken: String, queueURL: String, brazeApiKey:String)
 
 case class IdentityBrazeEmailData(externalId: String, emailAddress: String, templateId: String, customFields: Map[String, String])
 
 object IdentityBrazeEmailData {
   implicit val identityBrazeEmailDataDecoder: Decoder[IdentityBrazeEmailData] = deriveDecoder[IdentityBrazeEmailData]
 }
 
-case class BrazeResponse(msg: String)
+case class BrazeRecipient(external_user_id: String, trigger_properties: Option[Map[String, String]])
+
+object BrazeRecipient {
+  implicit val brazeRecipientEncoder: Encoder[BrazeRecipient] = deriveEncoder[BrazeRecipient]
+}
 
-case class Config(idapiHost: String, brazeApiHost: String, idapiAccessToken: String, queueURL: String)
+case class BrazeSendRequest(api_key: String,
+                            campaign_id: String,
+                            trigger_properties: Option[Map[String, String]],
+                            recipients: Seq[BrazeRecipient])
 
+object BrazeSendRequest {
+  implicit val BrazeSendRequestEncoder: Encoder[BrazeSendRequest] = deriveEncoder[BrazeSendRequest]
+  def apply(userId: String, apiKey: String, campaignId: String, customProperties: Map[String, String]): BrazeSendRequest = {
```

Comment:
not sure if a custom `apply()` method really adds anything?

Reply:
I guess we can keep the snake case syntax out of the code until the last moment this way?

Reply:
Sorry I misunderstood, have removed the nested class